CentralityiOS/CentralityiOS/ViewControllers/ModifyTaskModalViewController.m

```diff
@@ -137,5 +203,46 @@ - (IBAction)cancelAction:(id)sender {
     [self dismissViewControllerAnimated:YES completion:^{}];
 }
 
+- (BOOL)textFieldShouldBeginEditing:(UITextField *)textField
+{
+    IQKeyboardManager.sharedManager.keyboardDistanceFromTextField = kKeyboardDistanceFromTitleInput;
+    return YES;
+}
+
+-(BOOL)textFieldShouldEndEditing:(UITextField *)textField{
+    NSMutableAttributedString *stringWithSpaceAdded = [[NSMutableAttributedString alloc] initWithAttributedString:textField.attributedText];
+    [stringWithSpaceAdded appendAttributedString:[[NSMutableAttributedString alloc] initWithString:@" "]];
```

Comment:
is this the hack you put in place to get the highlighting working properly? you should probably have a comment near it so it's clear why you're adding the extra space

Reply:
Correct, this is the hack. No problem, I will add a comment here